src/main/scala/com/fulcrumgenomics/commons/util/StringUtil.scala

```diff
@@ -154,4 +154,29 @@ object StringUtil {
         throw new IllegalArgumentException("columnIt failed.  Did you forget to input an NxM table?")
     }
   }
+
+  /** Splits line on delimiter, placing up to `arr.length` values into `arr` and returning the number of values. */
```

Comment:
Can you please add a line to the docs stating 

>Multiple consecutive delimeters are treated as individual delimiters, delimiting empty fields.  E.g. with `deliminter=','` the string `a,b,,,c` will return ["a", "b", "", "", "c"]`.

This is important and we rely on it in multiple places, but other `split` operations (e.g. on String`) behave differently.